rpc/backend/blocks.go

```diff
@@ -160,7 +160,8 @@ func (b *Backend) GetBlockTransactionCount(block *tmrpctypes.ResultBlock) *hexut
 	}
 
 	ethMsgs := b.EthMsgsFromTendermintBlock(block, blockRes)
-	n := hexutil.Uint(len(ethMsgs))
+	ethMsgs2 := b.EthMsgsFromTendermintEndBlock(blockRes)
```

Comment:
what is difference between ethMsgs and ethMsgs2?

Reply:
ethMsg2 is from end block events